packages/@sanity/cli/src/actions/init-plugin/pluginTemplates.js

```diff
@@ -19,6 +19,11 @@ module.exports = [
     name: 'Chess board input component w/ block preview',
     url: 'https://github.com/sanity-io/plugin-template-chess-input/archive/master.zip',
   },
+  {
+    value: 'customInput',
+    name: 'A custom string input starting point',
+    url: 'https://github.com/sanity-io/sanity-plugin-custom-string/archive/refs/heads/main.zip',
+  },
```

Comment:
I would move this above the Chess Board input

```suggestion
  {
    value: 'customInput',
    name: 'A custom string input starting point',
    url: 'https://github.com/sanity-io/sanity-plugin-custom-string/archive/refs/heads/main.zip',
  },
```